test2.md

```diff
@@ -0,0 +1,5 @@
+# Additional test file created via dev-paul branch
+
+Here I have created an additional file for you to review which is not on main yet.
+
+Now I added some new lines to commit.
```

Comment:
hele slechte feature. Begin maar opnieuw

Reply:
sorry for the not so nice reply :)